webpack/config.js

```diff
@@ -17,7 +17,12 @@ if (!DEBUG) {
   plugins.push(new ManifestPlugin({
     fileName: assetMapFile
   }))
-  plugins.push(new webpack.optimize.UglifyJsPlugin({ minimize: true }))
+  plugins.push(new webpack.optimize.UglifyJsPlugin({
```

Comment:
Sort of. This PR required a lot of WebPack rebuilding, and these changes sped that up.